tee-worker/omni-executor/evm-aggregators-client/src/common.rs

```diff
@@ -0,0 +1,144 @@
+		map.insert(SWAP_NAME_PANCAKE_V2.to_string(), "pancake".to_string());
+		map.insert(SWAP_NAME_PANCAKE_V3.to_string(), "pancake-v3".to_string());
+		map
+	};
+	pub static ref INCH_DEX_IDS_MAP: HashMap<u64, HashMap<String, String>> = {
+		let mut map = HashMap::new();
+
+		let mut eth_map = HashMap::new();
+		eth_map.insert("UniswapV2".to_string(), "UNISWAP_V2".to_string());
+		eth_map.insert("UniswapV3".to_string(), "UNISWAP_V3".to_string());
+		eth_map.insert("PancakeV2".to_string(), "ETHEREUM_PANCAKESWAP_V2".to_string());
+		eth_map.insert("PancakeV3".to_string(), "PANCAKESWAP_V3".to_string());
+		map.insert(1, eth_map); // ETH Chain ID
+
+		let mut bsc_map = HashMap::new();
+		bsc_map.insert("UniswapV2".to_string(), "".to_string());
+		bsc_map.insert("UniswapV3".to_string(), "BSC_UNISWAP_V3".to_string());
+		bsc_map.insert("PancakeV2".to_string(), "PANCAKESWAP_V2".to_string());
+		bsc_map.insert("PancakeV3".to_string(), "BSC_PANCAKESWAP_V3".to_string());
+		map.insert(56, bsc_map); // BSC Chain ID
+
+		let mut base_map = HashMap::new();
+		base_map.insert("UniswapV2".to_string(), "BASE_UNISWAP_V2".to_string());
+		base_map.insert("UniswapV3".to_string(), "BASE_UNISWAP_V3".to_string());
+		base_map.insert("PancakeV2".to_string(), "BASE_PANCAKESWAP_V2".to_string());
+		base_map.insert("PancakeV3".to_string(), "BASE_PANCAKESWAP_V3".to_string());
+		map.insert(8453, base_map); // BASE Chain ID
+
+		map
+	};
+}
+
+lazy_static! {
+	pub static ref DECIMALS_TO_VALUE: HashMap<u8, i64> = {
+		let mut map = HashMap::new();
+		map.insert(0, 1);
+		map.insert(1, 10_i64.pow(1));
+		map.insert(2, 10_i64.pow(2));
+		map.insert(3, 10_i64.pow(3));
+		map.insert(4, 10_i64.pow(4));
+		map.insert(5, 10_i64.pow(5));
+		map.insert(6, 10_i64.pow(6));
+		map.insert(7, 10_i64.pow(7));
+		map.insert(8, 10_i64.pow(8));
+		map.insert(9, 10_i64.pow(9));
+		map.insert(10, 10_i64.pow(10));
+		map.insert(11, 10_i64.pow(11));
+		map.insert(12, 10_i64.pow(12));
+		map.insert(13, 10_i64.pow(13));
+		map.insert(14, 10_i64.pow(14));
+		map.insert(15, 10_i64.pow(15));
+		map.insert(16, 10_i64.pow(16));
+		map.insert(17, 10_i64.pow(17));
+		map.insert(18, 10_i64.pow(18));
+		map
+	};
+}
+
+pub const SWAP_NAME_UNIV2: &str = "Uniswap V2";
+pub const SWAP_NAME_UNIV3: &str = "Uniswap V3";
+pub const SWAP_NAME_PANCAKE_V2: &str = "PancakeSwap V2";
+pub const SWAP_NAME_PANCAKE_V3: &str = "PancakeSwap V3";
+pub const SWAP_NAME_FOUR_MEME_V2: &str = "FourMeme V2";
+
+lazy_static! {
+	pub static ref OKX_DEX_IDS_MAP: HashMap<String, String> = {
+		let mut map = HashMap::new();
+		map.insert(SWAP_NAME_UNIV2.to_string(), "OKX_UNIV2".to_string());
+		map.insert(SWAP_NAME_UNIV3.to_string(), "OKX_UNIV3".to_string());
+		map.insert(SWAP_NAME_PANCAKE_V2.to_string(), "OKX_PANCAKE_V2".to_string());
+		map.insert(SWAP_NAME_PANCAKE_V3.to_string(), "OKX_PANCAKE_V3".to_string());
+		map
+	};
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct CreateMarketTx {
```

Comment:
Unused fields:
  - omni_account: String
  - user_id: u64
  - user_wallet_id: u32
  - is_anti_mev: bool
  - is_auto_slippage: bool
  - out_decimal: u8
  - pair_addr: String
  - price: String
  - use_price_limit: bool
  - in_token_program: String
  - out_token_program: String

Does it make sense to keep them ?

Reply:
We don't need them for now, we may need it later, but no point keeping them for now, so removed.